### ISSUE TITLE
Wird angenommen, dass alle titlePage-Inhalte zentriert sind?

### ISSUE THREAD

**Issue by funkyfuture** [deutschestextarchiv/dtabf#32]
zumindest die renderings auf der webseite des DTA scheinen die in der beschreibung formulierte frage zu bejahen. bei den wenigen stichproben, die ich vorgenommen habe wird auch an keiner stelle zentrierter inhalt ausgezeichnet. in der dokumentation kann ich keinen entsprechenden hinweis auf eine solche annahme finden.

gleichzeitig gibt es phänomene, die dieser annahme nicht entsprechen, z.B.:

- [hier](http://www.deutschestextarchiv.de/book/view/meiselhess_weiberhass_1904?p=1) ist der autorinnenname linksbündig gesetzt
- [hier](http://www.deutschestextarchiv.de/book/view/lueders_interesse_1908?p=2) ist der titel als block gesetzt

**Comment by susannehaaf** [comment]
Ja, im DTA werden Informationen auf dem Titelblatt und die Überschriften grundsätzlich zentriert **dargestellt**. Es gibt aber keine Regel im DTABf, die von der genaueren Layout-**Auszeichnung** des Titelblatts abrät. Im Gegenteil: Typographische Informationen der Vorlage, die im Digitalen "tradiert" werden sollen, müssten auch ausgezeichnet werden (vgl. [hier](http://www.deutschestextarchiv.de/doku/basisformat/typogrAllg.html)). Die Darstellung ist davon unabhängig.
